### PR TITLE
Fix default value for fee limit

### PR DESCRIPTION
**Pull request by guggero**
## Change Description
Changes the default off-chain/routing fee to be amount dependent and no longer 100% of the payment amount in any case.

This only affects the older RPCs (`SendPayment`, `SendPaymentSync`, `QueryRoutes`) that had this default value of 100% set if the user didn't specify the limit explicitly.

The newer RPCs (`SendPaymentV2`) use a default value of 0 which forces the (API) user to set an explicit value.
We cannot do the same on the older RPCs as that would break backward compatibility of applications and annoy many users (because the symptom is a `NO_ROUTE` error instead of a more explicit `not enough fees` error).

### CODE REVIEW

File: cmd/lncli/cmd_payments.go [hunk 1]
```diff
@@ -7,6 +7,7 @@ import (
 	"encoding/hex"
 	"errors"
 	"fmt"
+	"github.com/lightningnetwork/lnd/lnwallet"
```
Comment (Crypt-iQ) [inline cmd/lncli/cmd_payments.go:10]: I think the linter wants this lower

**Review by Crypt-iQ — APPROVED** [review]
LGTM 🚡

I think the assumption here is if there is a 100sat payment and no route at 5sat, and instead one at 10sat, the user needs to set a fee-limit to control the actual fee.

File: cmd/lncli/cmd_payments.go (hunk 1 above)
Comment (guggero) [reply cmd/lncli/cmd_payments.go]: Oops, auto-inserted by my IDE. Fixed.

**Review by Roasbeef — APPROVED** [review]
LGTM 👑

**Pull request merged**